src/main/java/fly/wild/wizards/serverside/ServerConnnector.java

```diff
@@ -190,93 +117,208 @@ public boolean configureOneWayTLS () {
+			ModelNode undertow = client.execute(undertowBuilder.buildRequest());
+			this.logMessage.append("Updating Undertow\n");
+			this.logMessage.append(undertow.toString()+"\n");
 		} catch (IOException e) {
 			e.printStackTrace();
+		} catch (OperationFormatException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
 		}
 
 	}
 	
 	private void configureManagementInterface () {
+	
+		DefaultOperationRequestBuilder coreServiceSSLContextBuilder = new DefaultOperationRequestBuilder();
+		coreServiceSSLContextBuilder.setOperationName(Util.WRITE_ATTRIBUTE);
+		coreServiceSSLContextBuilder.addNode(Util.CORE_SERVICE,Util.MANAGEMENT);
+		coreServiceSSLContextBuilder.addNode(Util.MANAGEMENT_INTERFACE,Util.HTTP_INTERFACE);
+		coreServiceSSLContextBuilder.addProperty(Util.NAME,Util.SSL_CONTEXT);
+		coreServiceSSLContextBuilder.addProperty(Util.VALUE,this.genServerSSLContext);
 		
-		ModelNode coreServiceSSLContext = new ModelNode();
-		ModelNode coreServiceSecureSocketBinding = new ModelNode();
-		
-		coreServiceSSLContext.get(ClientConstants.OP).set(ClientConstants.WRITE_ATTRIBUTE_OPERATION);
-		coreServiceSSLContext.get(ClientConstants.OP_ADDR).add(STRING_CORE_SERVICE_ATTRIBUTE,STRING_CORE_SERVICE_VALUE);
-		coreServiceSSLContext.get(ClientConstants.OP_ADDR).add(STRING_MANAGEMENT_INTERFACE_ATTRIBUTE,STRING_MANAGEMENT_INTERFACE_VALUE);
-		coreServiceSSLContext.get(STRING_NAME_ATTRIBUTE).set("ssl-context");
-		coreServiceSSLContext.get("value").set(this.genServerSSLContext);
-		
-		coreServiceSecureSocketBinding.get(ClientConstants.OP).set(ClientConstants.WRITE_ATTRIBUTE_OPERATION);
-		coreServiceSecureSocketBinding.get(ClientConstants.OP_ADDR).add("core-service","management");
-		coreServiceSecureSocketBinding.get(ClientConstants.OP_ADDR).add("management-interface","http-interface");
-		
-		coreServiceSecureSocketBinding.get("name").set("secure-socket-binding");
-		coreServiceSecureSocketBinding.get("value").set("management-https");
-		
+		DefaultOperationRequestBuilder coreServiceSecureSocketBindingBuilder = new DefaultOperationRequestBuilder();
+		coreServiceSecureSocketBindingBuilder.setOperationName(Util.WRITE_ATTRIBUTE);
+		coreServiceSecureSocketBindingBuilder.addNode(Util.CORE_SERVICE,Util.MANAGEMENT);
+		coreServiceSecureSocketBindingBuilder.addNode(Util.MANAGEMENT_INTERFACE,Util.HTTP_INTERFACE);
+		coreServiceSecureSocketBindingBuilder.addProperty(Util.NAME,Util.SECURE_SOCKET_BINDING);
+		coreServiceSecureSocketBindingBuilder.addProperty(Util.VALUE,Util.MANAGEMENT_HTTPS);
+
 		try {
-			this.client = ModelControllerClient.Factory.create (InetAddress.getByName(this.ipAddress), 
-					DMR_PORT);
 			
-			client.execute(new OperationBuilder(coreServiceSSLContext).build());
-			client.execute(new OperationBuilder(coreServiceSecureSocketBinding).build());
-		} catch (IOException e) {
+			ModelControllerClient client = ModelControllerClientFactory.CUSTOM.getClient(connectionAddress, null, false, null, false, 0, null, null, null);
```

Comment:
just a minor, you can add a new method that creates a client and use it here and above